site/docs/encoding/axis.md

```diff
@@ -46,33 +44,38 @@ Besides `axis` property of a field definition, the configuration object ([`confi
 }
 ```
 
-To customize axis, an `axis` object in [an encoding channel's definition](encoding.html) can contain the following groups of properties:
+{:#properties}
+
+To customize axis, an `axis` object in [an encoding channel's definition](encoding.html) can contain the following properties:
 
 {:#general}
 ### General
 
-{% include table.html props="bandPosition,domain,domainColor,domainWidth,grid,gridColor,gridDash,gridOpacity,gridWidth,maxExtent,minExtent,orient,offset,position,zindex" source="Axis" %}
+{% include table.html props="bandPosition,maxExtent,minExtent,orient,offset,position,zindex" source="Axis" %}
 
 __See also:__ [General Axis Config](#general-config).
 
 {:#labels}
 ### Labels
 
-{% include table.html props="format,labels,labelAlign,labelAngle,labelBaseline,labelColor,labelFont,labelFontSize,labelLimit,labelOverlap,labelPadding" source= "Axis" %}
+{% include table.html props="format,labels,labelAlign,labelAngle,labelBaseline,labelBound,labelColor,labelFlush,labelFlushOffset,labelFont,labelFontSize,labelFontWeight,labelLimit,labelOpacity,labelOverlap,labelPadding" source= "Axis" %}
 
-__See also:__ [Axis Labels Config](#labels-config) and [`guide-label` style config](mark.html#style-config) (common styles for axis, [legend](legend.html), and [header](facet.html#header) labels).
+__See also:__  [`guide-label` style config](mark.html#style-config) (common styles for axis, [legend](legend.html), and [header](facet.html#header) labels).
 
 {:#ticks}
 ### Ticks
 
-{% include table.html props="ticks,tickColor,tickCount,tickRound,tickSize,tickStep,tickWidth,values" source="Axis" %}
+{% include table.html props="tickColor,tickCount,tickExtra,tickOffset,tickOpacity,tickRound,ticks,tickSize,tickStep,tickWidth,values" source="Axis" %}
+
+{:#domain}
+### Domain
```

Comment:
Let's add this section above labels instead.  (Alphabetical order.)

Reply:
doen